SphereEversion/ToMathlib/Geometry/Manifold/Immersion.lean

```diff
@@ -56,7 +56,7 @@ structure Immersion (f : M → M') (n : WithTop ℕ∞) : Prop where
   diff_injective : ∀ p, Injective (mfderiv I I' f p)
 
 /-- An injective `C^n` immersion -/
-structure InjImmersion (f : M → M') (n : WithTop ℕ∞) extends Immersion I I' f n : Prop where
+structure InjImmersion (f : M → M') (n : WithTop ℕ∞) : Prop extends Immersion I I' f n where
```

Comment:
Is this necessary? I don’t mind the change in any case, I’m only curious.

Reply:
Yeah, this was a change from https://github.com/leanprover/lean4/pull/7100